src/QMCTools/ppconvert/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# this new link path policy has fixed ldpath problems
+# experienced with out of dyld_load_path libraries on
+# OSX without bad effect on other platforms.
+cmake_policy(SET CMP0003 NEW)
+
+# in this directory and below remove the -DNDEBUG flag
+string(REPLACE "-DNDEBUG" "" CMAKE_CXX_FLAGS_RELEASE ${CMAKE_CXX_FLAGS_RELEASE})
```

Comment:
If this CMAKE_CXX_FLAGS change only affects locally, why not apply it on CMAKE_CXX_FLAGS instead of just release. Someone like me, maybe build release with debug case.

Reply:
Good point

Reply:
I think this is the last thing.

Reply:
Except it doesn't work as CMAKE_CXX_FLAGS doesn't contain -DNDEBUG only release has that by default.  In build config release you get CMAKE_CXX_FLAGS + CMAKE_CXX_FLAGS_RELEASE. You don't get CMAKE_CXX_FLAGS += CMAKE_CXX_FLAGS_RELEASE.  So it can't be removed from there. I think just protecting against the default config flags is sufficient. Someone who is messing with flags by hand and has ppconvert break is likely to take a look in here.

Reply:
I see.
Could you also do the same removal for CMAKE_CXX_FLAGS_RELWITHDEBINFO which also has -DNDEBUG.
Thank you.